src/utils/storage/update_handlers/v0.14.2.ts

```diff
@@ -0,0 +1,23 @@
+import { TOKENS } from "../../../constants/tokens";
+import { StorageData } from "../../../interfaces/storage"
+
+export default (storage: any) : StorageData => ({
+    ...storage,
+    valut: {
+        ...storage.vault,
+        wallets: storage.value.wallets.map((wallet) => ({
+            ...wallet,
+            assets: wallet.assets.reduce((acum, asset) => ({...acum, [asset.canisterId]: {
+                balance: '0',
+                token: {
+                    name: asset.name,
+                    symbol: asset.symbol,
+                    canisterId: asset.canisterId,
+                    standard:  wallet.registeredTokens[asset.canisterId]?.standard || TOKENS[asset.symbol].standard,
+                    decimals: wallet.registeredTokens[asset.canisterId]?.decimals || TOKENS[asset.symbol].decimals,
+                    color: wallet.registeredTokens[asset.canisterId]?.color
+                }
+            }}), {})
+        }))
+    }
+});
```

Comment:
time to test this out lol

Reply:
![image](https://user-images.githubusercontent.com/19255688/150539113-3fa70b8a-980a-4d66-91c0-ef73f089ab1a.png)